### ISSUE TITLE
File editor stopped working in 7.1.2

### ISSUE THREAD

**Issue by jpalo** [pnp/sp-editor#75]
# Steps
1. Start SP Editor
2. Go to File Editor
3. Click on "_catalogs"

Error:
![image](https://github.com/user-attachments/assets/c1bb404f-5c20-4843-a379-1f9027f49a5c)

SP 2016, Edge 130.0.2849.27, SP Editor 7.1.2

SP Editor 5.6.17 works fine in this same environment.

**Comment by tavikukko** [comment]
Thanks for the issue @jpalo !
Could you give some more information about the environment, looks like the path contains some interesting characters.

**Comment by jpalo** [comment]
It's the error message that has weird characters as it is in Finnish. Url itself doesn't have any weird chars.

**Comment by tavikukko** [comment]
Ok thanks. Could you please check the network tab, and see what is the url that SP Editor is calling and is failing?

**Comment by tavikukko** [comment]
hmm, looks like sp2016 does not have _api/web/getFolderByServerRelative**Path** endpoint.. it has _api/web/getFolderByServerRelative**Url**, which seems like is not part of PnPJS anymore.

**Comment by tavikukko** [comment]
I need to add custom behavior for PnPJs to change the path, seems like it also works SP online.
```
    /*** custom behavior ***/
    const onpremfix = () => {
      return (instance: Queryable.Queryable) => {
        instance.on.pre.prepend(async (url, init, result) => {
          return [
            url
              .replace('getFileByServerRelativePath(decodedUrl=', 'getFileByServerRelativeUrl(')
              .replace('getFolderByServerRelativePath(decodedUrl=', 'getFolderByServerRelativeUrl('),
            init,
            result,
          ];
        });
        return instance;
      };
    };
```
Currently working on the next release of SP Editor, with some enhancements for File Editor, so will add this fix part of it.

**Comment by tavikukko** [comment]
Fixed in the latest version 7.2.0
Could you test when possible, thanks @jpalo !

**Comment by jpalo** [comment]
Works great, thanks!
